core/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -300,6 +300,35 @@ public synchronized long getUserId() throws IOException {
     return null;
   }
 
+  public synchronized long getUsedBytes() throws IOException {
```

Comment:
That is a significant amount of code change (thrift server, interface, etc) to do some small things, not sure if that is necessary and worthwhile.  Let me know if you want to take this change, and if not then it is fine, I will close this PR.